SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -300,12 +348,6 @@ void ActorSystem::HandleActorAuthority(const Worker_EntityId EntityId, const Wor
 
 	if (NetDriver->IsServer())
 	{
-		// TODO UNR-955 - Remove this once batch reservation of EntityIds are in.
-		if (Authority == WORKER_AUTHORITY_AUTHORITATIVE)
```

Comment:
This removed just due to the reference to the ticket above?

Reply:
I think this was removed because I'm pretty sure the code path can't be hit. Will double check, maybe this was overzealous

Reply:
going to add this back for the reasons discussed in the other thread (actions taken between actor creation and entity creation). Not sure if the mentioned ticket is relevant anymore https://improbableio.atlassian.net/browse/UNR-955, so will drop reference to that and add a comment about what this is now doing